core/pyproject.toml

```diff
@@ -1,2 +1,34 @@
-[tool.black]
+[tool.ruff]
```

Comment:
@Innixma @shchur would this change detect all the src paths for all modules after pip install ?
as that was the problem earlier

Reply:
Probably not. But we can check after. My current plan is to remove the `pyproject.toml` files except for the root one after we converge on 119 line-length.